distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Display_Helpers.enso

```diff
@@ -87,3 +85,24 @@ pad txt len =
 ansi_bold : Boolean -> Text -> Text
 ansi_bold enabled txt =
     if enabled && (Java_System.console != Nothing) then '\e[1m' + txt + '\e[m' else txt
+
+## PRIVATE
+   Gets a textual representation of the item at position `ix` in `storage`.
+private _get_item_as_text storage ix =
+    item = storage.getItemBoxed ix
+    ## Special handling for display of Text to avoid quotes when not necessary.
+    case item of
+        _ : Text -> _normalize_text_for_display item
+        Nothing -> "Nothing"
+        _ -> item.pretty
```

Comment:
```suggestion
        _ -> item.pretty
```
`.pretty` handles the nothing case too, so we don't need the special case anymore